release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1122,9 +1123,9 @@ For more information, see xref:../updating/updating-restricted-network-cluster/r
 
 [discrete]
 [id="ocp-4-13-ocm-nodeip-configuration-vsphere"]
-=== The `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster
+=== The nodeip-configuration service is now enabled on a vSphere user-provisioned infrastructure cluster
 
-In {product-title} {product-version}, the `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster. This service determines the network interface controller (NIC) that {product-title} uses for communication with the Kubernetes API server when the node boots. In rare circumstances, the service might select an incorrect node IP after an upgrade. If this happens, you can use the `NODEIP_HINT` feature to restore the original node IP, as described in the "Troubleshooting network issues" section of the documentation.
+In {product-title} {product-version}, the `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster. This service determines the network interface controller (NIC) that {product-title} uses for communication with the Kubernetes API server when the node boots. In rare circumstances, the service might select an incorrect node IP after an upgrade. If this happens, you can use the `NODEIP_HINT` feature to restore the original node IP. See xref:../support/troubleshooting/troubleshooting-network-issues.adoc#troubleshooting-network-issues[Troubleshooting network issues].
```

Comment:
```suggestion
In {product-title} {product-version}, the `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster. This service determines the network interface controller (NIC) that {product-title} uses for communication with the Kubernetes API server when the node boots. In rare circumstances, the service might select an incorrect node IP after an upgrade. If this happens, you can use the `NODEIP_HINT` feature to restore the original node IP. See xref:../support/troubleshooting/troubleshooting-network-issues.adoc#troubleshooting-network-issues[Troubleshooting network issues].
```
```suggestion
In {product-title} {product-version}, the `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster. This service determines the network interface controller (NIC) that {product-title} uses for communication with the Kubernetes API server when the node boots. In rare circumstances, the service might select an incorrect node IP after an upgrade. If this happens, you can use the `NODEIP_HINT` feature to restore the original node IP. See xref:../support/troubleshooting/troubleshooting-network-issues.adoc_troubleshooting-network-issues[Troubleshooting network issues].
```

Your xref is not rendering properly in the preview. Give this a try and see if it works.